src/frontend/packages/lib_video/src/components/common/DashboardControlPane/index.tsx

```diff
@@ -63,11 +74,21 @@ export const DashboardControlPane = ({ isLive }: DashboarControlPaneParams) => {
           <Tab title={intl.formatMessage(messages.titleConfiguration)}>
             <VideoWidgetProvider isLive={isLive} isTeacher />
           </Tab>
-          <Tab title={intl.formatMessage(messages.titleAttendance)}>
-            <DashboardLiveTabAttendance />
-          </Tab>
+          {hasStats ? (
+            <Tab title={intl.formatMessage(messages.titleStats)}>
+              <DashboardTabStatistics />
+            </Tab>
+          ) : (
+            <Tab title={intl.formatMessage(messages.titleAttendance)}>
+              <DashboardLiveTabAttendance />
+            </Tab>
+          )}
```

Comment:
I'm not a big fan of controlling attendance visibility with a props called `hasStats`